### PR TITLE
Contact Form & Footer add

### PR DESCRIPTION
**Pull request by Kambuchi**
_**Contact**_

--Name form add
--Email form add
--Text area add

_**Footer**_

--Link add
--Network icons add

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -123,6 +123,46 @@ <h3>Skills</h3>
...
+            <div class="contact-container">
+                <h2>Get started</h2>
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+                sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+                <form action="https://formspree.io/f/mqkwbzjb" method="post">
+                    <ul class="form-fields">
+                     <li class="float-container">
+                       <label for="name">Your name</label>
+                       <input type="text" id="name" name="user_name" placeholder="Jhon Doe" maxlength="30" required>
+                     </li>
```
Comment (henatan99) [inline index.html]: - [ ] No `ul` , `li` tags are required inside a `form`. Kindly remove all.
- [ ] No `label` tag is required here as you are using `placholder`.

File: style.css [hunk 2]
```diff
@@ -219,3 +218,133 @@ nav li {
   background-position: right bottom;
   background-size: 32vw;
 }
+
+.contact {
```
Comment (henatan99) [inline style.css:223]: - This element should align its elements to the center. Kindly see the screenshot image of your page and implement changes. 
Screenshot this

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi Kambuchi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi @Kambuchi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**